lib/tasks/export_local_transactions.rake

```diff
@@ -0,0 +1,15 @@
+desc "Export Local Transactions' slugs, names, LGSL and optional LGIL codes as CSV"
+
+task :export_local_transactions => :environment do
+  require "csv"
+
+  csv_string = CSV.generate do |csv|
+    csv << ["slug","lgsl","lgil","title"]
+
+    Edition.where(_type: "LocalTransactionEdition").each do |lte|
```

Comment:
This is exporting all editions, not only the currently published one, so services are duplicated in the output. We'll need to update this to include only the currently published edition for each service.